rockcraft.yaml

```diff
@@ -1,7 +1,7 @@
 name: kratos
 base: bare
 build-base: ubuntu@22.04
-version: "1.1.0"
+version: "1.3.0"
```

Comment:
[1.3.1](https://github.com/ory/kratos/releases/tag/v1.3.1) is available, should we use that?

Reply:
I d say we leave this and make another pr to test the machinery